eland/operations.py

```diff
@@ -1273,6 +1273,8 @@ def to_json(  # type: ignore
             for i, df in enumerate(
                 self.search_yield_pandas_dataframes(query_compiler=query_compiler)
             ):
+                if pd.__version__.split(".")[0] == "2":
```

Comment:
Let's use a helper method instead of always looking at `pd.__version__`. Additionally, the logic should be "if it's not Pandas 1", which will help with the [upcoming Pandas 3](https://github.com/pandas-dev/pandas/milestone/102).
```suggestion
                if not PANDAS_15:
```

This comment applies to all the places where you currently read `pd.__version__`. This will help a lot when we drop Pandas 1.5. (And is how I spotted the removeprefix change in https://github.com/elastic/eland/pull/743/files#diff-d69877df0768b385c97796c7a19ab8b0582f2ee99dbca40ca78ed7fc68bf9ff0.)

Reply:
I found a global in `eland.common` that I now use in all places that did `pd.__version__.split('.')`.